XRTK-Core/Packages/com.xrtk.core/Editor/PackageInstaller.cs

```diff
@@ -193,6 +199,12 @@ private static string CopyAsset(this string rootPath, string sourceAssetPath, st
         /// <param name="rootProfile">The root profile to install the </param>
         public static void InstallConfiguration(MixedRealityPlatformServiceConfigurationProfile platformConfigurationProfile, MixedRealityToolkitRootProfile rootProfile)
         {
+            try
+            {
+                Selection.activeGameObject = null;
```

Comment:
I'd like to see this be `Selection.activeObject` in case we have a profile selected.

Reply:
Updated